cdap-docs/_common/common-build.sh

```diff
@@ -358,7 +367,57 @@ function display_version() {
   echo "PROJECT_SHORT_VERSION: ${PROJECT_SHORT_VERSION}"
   echo "GIT_BRANCH: ${GIT_BRANCH}"
   echo "GIT_BRANCH_TYPE: ${GIT_BRANCH_TYPE}"
-  echo ""
+}
+
+function set_message_file() {
+  TMP_MESSAGE_FILE="/tmp/$(basename $0).$$.tmp"
+  export TMP_MESSAGE_FILE
+}
+
+function clear_message() {
+  MESSAGE=""
+  set_message_file
+}
+
+function set_message() {
+  if [ "${MESSAGE}" == "" ]; then
+    MESSAGE="${1}"
+  else
+    MESSAGE="${MESSAGE}\n\n${1}"
```

Comment:
Actually, it may be better to just keep the input as an array and append it to the previous array, assuming you make `MESSAGE` an array, versus a string as well as `result_message` above.